server/utils/request.ts

```diff
@@ -0,0 +1,40 @@
+import { RequestInterface } from '../models/requestModel'
+import { ServerResponseInterface } from '../graphql/serverResponse'
+import { UserInputError } from 'apollo-server-errors'
+
+import { updateRequestTypeHelper } from './requestType'
+
+const filterOpenRequests = (requests: Array<RequestInterface> ) => {
+  return requests.filter(request => request.fulfilled === false && request.deleted === false)
+}
+
+const filterFulfilledRequests = (requests: Array<RequestInterface> ) => {
+  return requests.filter(request => request.fulfilled === false && request.deleted === false)
```

Comment:
Shouldn't request.fufilled == true if we are filtering by fulfilled requests?

Reply:
Updated.